### ISSUE TITLE
Bitcoin wallet service - installation help

### ISSUE THREAD

**Issue by iangregsondev** [bitpay/bitcore#2734]
Hi,

I wanted to give the bitcoin wallet service a try but I was a little confused about the way to go about it.

I notice its a package, does it have a dependency on running the bicoin node that you provide ?

I mean how would it broadcast a transaction, does this need the bitcoin node?

So the flow would be, running your bitcoin node (details on the readme) and then the bitcoin wallet service uses this node ?

**Comment by vishvaultbank** [comment]
Yeah...You need to have a running `bitcoind` and then you can connect using configuration options in `bitcore.config.json` file

**Comment by iangregsondev** [comment]
Thanks @vishvaultbank , so the only dependency is basically bitcoind, there is no dependency on any other bitcore packages ?

So I would just run what's below the wallet services directory ?

**Comment by SwenVanZanten** [comment]
You also need a bitcore-node which is connected to the bitcoind. Which is explained here:
https://github.com/bitpay/bitcore

Then you can connect your bws to bn.
